src/measure/angular_rdf.cu

```diff
@@ -0,0 +1,730 @@
+/*
+    Copyright 2017 Zheyong Fan and GPUMD development team
+    This file is part of GPUMD.
+    GPUMD is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    GPUMD is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License
+    along with GPUMD.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/*-----------------------------------------------------------------------------------------------100
+Calculate:
+    Angular-dependent Radial distribution function (AngularRDF)
+--------------------------------------------------------------------------------------------------*/
+
+#include "angular_rdf.cuh"
+#include "force/neighbor.cuh"
+#include "model/atom.cuh"
+#include "model/box.cuh"
+#include "model/group.cuh"
+#include "parse_utilities.cuh"
+#include "utilities/common.cuh"
+#include "utilities/error.cuh"
+#include "utilities/gpu_macro.cuh"
+#include "utilities/read_file.cuh"
+#include <cstring>
```

Comment:
You need to add `#include <cmath>` here as you used `M_PI`. Linux might have automatically included this, but not in Windows.